test/docs/testplans/vnet.md

```diff
@@ -0,0 +1,156 @@
+|  8  | Verify route action ROUTE_VNET_DIRECT with CA to PA mappings | `Vnet2VnetOutboundDstVnetIdRouteVnetDirectSinglePortTest.`<br/> `vnet2VnetOutboundDstVnetIdTrueTest`<br/> `vnet2VnetOutboundDstVnetIdFalseTest`<br><br>`Vnet2VnetOutboundDstVnetIdRouteVnetDirectTwoPortsTest.`<br/> `vnet2VnetOutboundDstVnetIdTrueTest`<br/> `vnet2VnetOutboundDstVnetIdFalseTest`                                     | Creates single ENI with two outbound routing entries (with SAI_OUTBOUND_ROUTING_ENTRY_ACTION_ROUTE_VNET_DIRECT action) and ca_to_pa entries (with use_dst_vnet_vni attribute True and False values).<br/> Verifies configuration with bidirectional VXLAN TCP traffic.<br/>1. With underlay route<br/>2. Without underlay route  |
+| 9-10 | Use same CA prefixes in different outbound routing tables (different ENIs)    | `Vnet2VnetOutboundMultipleEniSameIpPrefixSinglePortTest.`<br/> `outboundEni0Test`<br/> `outboundEni1Test`<br/> `outboundEni2Test`<br><br>`Vnet2VnetOutboundMultipleEniSameIpPrefixTwoPortsTest.`<br/> `outboundEni0Test`<br/> `outboundEni1Test`<br/> `outboundEni2Test`                                                                 | Creates three ENI with the same Customer and Physical IP addresses but with different MACs and VNIs.<br/> Verifies configuration with bidirectional VXLAN TCP traffic.<br/>1. With underlay route<br/>2. Without underlay route |
+| 11-12 | Use multiple overlapping routing prefixes in the same outbound routing table. | `Vnet2VnetOutboundSingleEniMultipleIpPrefixSinglePortTest.`<br/> `singleEniToOutboundVm1Test`<br/> `singleEniToOutboundVm2Test`<br/> `singleEniToOutboundVm3Test`<br><br>`Vnet2VnetOutboundSingleEniMultipleIpPrefixTwoPortsTest.`<br/> `singleEniToOutboundVm1Test`<br/> `singleEniToOutboundVm2Test`<br/> `singleEniToOutboundVm3Test` | Creates single ENI with three outbound routing entries with overlapping IP prefixes (ENI 9.0.0.1 <--> 10.5.4.4/8, 10.0.1.2/24, 10.1.1.1/32).<br/> Verifies configuration with bidirectional VXLAN TCP traffic.<br/>1. With underlay route<br/>2. Without underlay route |
+| 13-14 | Use same prefixes in CA and PA networks. | `Vnet2VnetOutboundSameCaPaIpPrefixesSinglePortTest.`<br/> `vnet2VnetOutboundRouteVnetTest`<br><br>`Vnet2VnetOutboundSameCaPaIpPrefixesTwoPortsTest.`<br/> `vnet2VnetOutboundRouteVnetTest`                                                                                                                                               | Creates single ENI with the same Customer and Physical IP address outbound configuration.<br/> Verifies configuration with bidirectional VXLAN TCP traffic.<br/>1. With underlay route<br/>2. Without underlay route |
+
+
+Original table [link](https://github.com/Azure/DASH/blob/main/documentation/general/design/sdn-features-packet-transforms.md#routing-routes-and-route-action).
```

Comment:
Link is broken, `design` was removed from path quite some time ago.
Also, change `Azure` to `sonic-net`